kyma/extensions/configuration-cluster/openid-connect.yaml

```diff
@@ -49,34 +52,23 @@ data:
       source: spec.clientID
   form: |-
     - path: spec.issuerURL
-      visibility: $not((metadata.labels."operator.kyma-project.io/managed-by") = "infrastructure-manager")
-      overwrite: false
+      widget: Text
       placeholder: Issuer
     - path: spec.usernameClaim
-      visibility: $not((metadata.labels."operator.kyma-project.io/managed-by") = "infrastructure-manager")
-      overwrite: false
       widget: Text
       placeholder: Username Claim
     - path: spec.groupsClaim
-      visibility: $not((metadata.labels."operator.kyma-project.io/managed-by") = "infrastructure-manager")
-      overwrite: false
       widget: Text
       placeholder: Groups Claim
     - path: spec.clientID
-      visibility: $not((metadata.labels."operator.kyma-project.io/managed-by") = "infrastructure-manager")
-      overwrite: false
       widget: Text
       placeholder: Client ID
     - path: spec.jwks.distributedClaims
-      visibility: $not((metadata.labels."operator.kyma-project.io/managed-by") = "infrastructure-manager")
-      overwrite: false
       enum:
       - true
       - false
     - path: spec.supportedSigningAlgs
-      visibility: $not((metadata.labels."operator.kyma-project.io/managed-by") = "infrastructure-manager")
-      overwrite: false
       widget: SimpleList
-      defaultExpanded: true
       children:
         - path: '[]'
+          description: ''
```

Comment:
Simple and nice solution, to be honest I didn't remember it will work like this, but please open an bug issue for misalignment of the description in SimpleLists.